types/dropzone/index.d.ts

```diff
@@ -228,6 +234,14 @@ declare class Dropzone {
         callback?: (...args: any[]) => void,
     ): any;
 
+    displayExistingFile(
+        mockFile: Dropzone.DropzoneMockFile,
+        imageUrl: string,
+        callback?: (...args: any[]) => void,
```

Comment:
I think we can just have `() => void` here
Since it never calls the callback with any argument

https://gitlab.com/meno/dropzone/-/blob/v5.7.0/src/dropzone.js#L1685-1701

Reply:
Good point, I'll apply the changes.